### PR TITLE
stable: rollout 32.20200629.3.0

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: updates/stable.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
   "releases": [
     {
-      "version": "32.20200601.3.0",
+      "version": "32.20200615.3.0",
       "metadata": {
+        "barrier": {
+          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/484"
+        },
         "rollout": {
           "start_percentage": 1.0
```
Comment (dustymabe) [inline updates/stable.json:14]: Do we need to leave the rollout/start_percentage information here? I'm comparing it to `next.json` and I don't see that in there.

File: updates/stable.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
     {
-      "version": "32.20200615.3.0",
+      "version": "32.20200629.3.0",
       "metadata": {
-        "barrier": {
-          "reason": "https://github.com/coreos/fedora-coreos-tracker/issues/484"
-        },
         "rollout": {
-          "start_epoch": 1593540000,
+          "start_epoch": 1568385000,
```
Comment (dustymabe) [inline updates/stable.json:22]: Fix?

```
stable rollout of 32.20200629.3.0 scheduled for 2019-09-13 14:30:00 UTC for 4320m (72.0h)
```

Comment (bgilbert) [reply updates/stable.json:22]: Whoops.

File: updates/stable.json (hunk 1 above)
Comment (bgilbert) [reply updates/stable.json:14]: Maybe not.  Let's try it.

**Comment by dustymabe** [comment]
Rollout time looks good!

```
$ date -u -d @1594650600
Mon 13 Jul 2020 02:30:00 PM UTC
$ date -d @1594650600
Mon 13 Jul 2020 10:30:00 AM EDT
```

File: updates/stable.json (hunk 1 above)
Comment (bgilbert) [reply updates/stable.json]: Yup, looks good without.

**Pull request merged**